examples/Quadrotor/office.urdf

```diff
@@ -131,30 +131,6 @@
     </collision>
    </link>
 
-    <link name="tableLegLL">
```

Comment:
this was repeated text in the file.  this (and all of the above changes) were simply fixes to the urdfs.